### PR TITLE
EPMRPP-33458 React. Dashboard. Widget. Launches table

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    formatMessage = props.intl.formatMessage;
+    this.columns = this.getColumns();
+  }
+
+  getFields = () => {
+    const { widget } = this.props;
+    const fields = [];
+
+    const fieldsFormProps = widget.content_parameters.content_fields;
+    fieldsFormProps.forEach((item) => {
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: As a variant. Thanks. Done

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  const itemValue = value.values[name];
+
+  return (
+    <div className={cx('defect-col', className)}>
+      <div className={cx('desktop-block')}>
+        {!!Number(itemValue) &&
+          (name === STATS_TI ? (
+            <ToInvestigateStatistics
+              value={{
+                [nameConfig[3]]: itemValue,
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Please define corresponding variables.

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+                total: itemValue,
+              }}
+              itemId={Number(value.id)}
+            />
+          ) : (
+            <DefectStatistics
+              type={nameConfig[2]}
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Please define corresponding variables.

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+              data={{
+                [nameConfig[3]]: itemValue,
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Please define corresponding variables.

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Fixed

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx (hunk 3 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Fixed

File: app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx (hunk 4 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardPage/widgets/launchesTable/launchesTable.jsx]: Fixed

**Pull request merged**